includes/discovery/applications.inc.php

```diff
@@ -77,14 +78,26 @@
         if (in_array($app, $enabled_apps)) {
             echo '.';
         } else {
-            dbInsert([
-                'device_id' => $device['device_id'],
-                'app_type' => $app,
-                'discovered' => 1,
-                'app_status' => '',
-                'app_instance' => '',
-            ], 'applications');
-
+            Application::upsert(
```

Comment:
i still don't want us to use upsert for a single row.

Reply:
@murrant For some reason that works fine when used but breaks CI. That is why I revered it in [03c34ec](https://github.com/librenms/librenms/pull/15270/commits/03c34ec10d776c06365481d47f67a06f9c57b7b7).

Reply:
Actually I take that back. Was not setting discovered.